stock_lot_analytic/models/stock_move_line.py

```diff
@@ -13,3 +13,16 @@ def _get_value_production_lot(self):
             "analytic_distribution"
         ] = self.move_id.purchase_line_id.analytic_distribution
         return res
+
+    def _action_done(self):
+        res = super()._action_done()
+        self = self.exists()
+        for ml in self:
+            if not ml.lot_id:
+                continue
+            purchase_line = ml.move_id.purchase_line_id
+            if purchase_line.analytic_distribution != ml.lot_id.analytic_distribution:
+                ml.lot_id.write(
+                    {"analytic_distribution": purchase_line.analytic_distribution}
```

Comment:
@nobuQuartile I forget why we are taking the value from purchase line instead of stock move line. Can you do a quick comparison of the values in copy or test database to identify where there are any discrepancies?